src/Idler/MainWindow.xaml.cs

```diff
@@ -394,5 +447,24 @@ private void RestoreWindowPosition()
             this.Left = Settings.Default.MainWindowLeft;
             this.WindowState = Settings.Default.MainWindowMaximized ? WindowState.Maximized : WindowState.Normal;
         }
+
+        private void FetchMonthlyTotalEffort(int month, int year)
+        {
+            if (!Settings.Default.IsHighlightingEnabled || Settings.Default.DailyWorkLoad == 0)
+            {
+                if (this.MonthlyTotalEffort.Count == 0)
+                {
+                    this.MonthlyTotalEffort = new Dictionary<DateTime, decimal>(0);
+                }
+                return;
+            }
+
+            this.SafeAsyncCall(
```

Comment:
`SafeAsyncCall` sets property `IsBusy` and I don't think we want to do it when we load days for highlighting

Reply:
implemented logic to skip setting property `isBusy`